src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateRange/index.tsx

```diff
@@ -197,19 +200,27 @@ class DateRange extends React.Component<Props, State> {
 
     return (
       <div className={className} data-test-id="date-range">
-        <StyledDateRangePicker
-          rangeColors={[theme.purple300]}
-          ranges={[
-            {
-              startDate: moment(start).local().toDate(),
-              endDate: moment(end).local().toDate(),
-              key: 'selection',
-            },
-          ]}
-          minDate={minDate}
-          maxDate={maxDate}
-          onChange={this.handleSelectDateRange}
-        />
+        <React.Suspense
```

Comment:
Would be nice if we could get this module loaded when the date range is first opened. That might save some spinner time, as the module could be loaded while the user is moving down to the 'absolute date' option.